test/test_cotrip.py

```diff
@@ -265,7 +311,7 @@ def test_parse_alert_from_street_success():
             "end_date_accuracy": "estimated",
             "beginning_accuracy": "estimated",
             "ending_accuracy": "estimated",
-            "road_name": "I-25",
+            "road_names": ['I-25'],
```

Comment:
Since we're adding potentially two road names in here now, can we have a unit test showing that functionality?

Reply:
Added unit test for that